packages/webapp/pages/onboarding.tsx

```diff
@@ -355,11 +357,11 @@ export function OnboardPage(): ReactElement {
 
     const params = new URLSearchParams(window.location.search);
     const afterAuth = params.get(AFTER_AUTH_PARAM);
+    const query = onboardingPapercut ? undefined : { ua: 'true' };
```

Comment:
Why do we remove this for the experiment, I don't see what papercut that solves?

Reply:
They want the user to focus on the app and use the features immediately rather than answer our questions/etc - at least the way I see it. Though the experiment is only a guardrail and they just want this be removed.

Reply:
Oh right UA is the one that shows the survey 😅
Sorry my bad, thought it was something marketing cookie related